### PR TITLE
Mccalluc/docs

### PR DESCRIPTION
**Pull request by mccalluc**
Refactoring to support running multiple Dash apps simultaneously: One will be `vis`, one will be `help`.

### CODE REVIEW

File: context/app/vis/callbacks.py [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
 from app.utils.cluster import cluster
 from app.utils.frames import sort_by_variance
+from app.vis.condition_callbacks import VisConditionCallbacks
+from app.vis.gene_callbacks import VisGeneCallbacks
 
 
-class AppCallbacks(AppGeneCallbacks, AppConditionCallbacks):
+class VisCallbacks(VisGeneCallbacks, VisConditionCallbacks):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
```
Comment (scottx611x) [inline context/app/vis/callbacks.py]: I believe `super` can only resolve the `__init__` here for `VisGeneCallbacks`, I've run into something similar in Refinery.

From https://stackoverflow.com/a/3810460
```
super() will only ever resolve a single class type for a given method, so if you're inheriting from multiple classes and want to call the method in both of them, you'll need to do it explicitly.
```

Comment (mccalluc) [reply context/app/vis/callbacks.py]: Oh, that's good to know. Thanks. The primary motivation behind all of this is to actually have two Dash apps (one for vis, one for online docs) running under a single Flask, so I wanted to create a new level with subpackages to keep the two separate.

Comment (mccalluc) [reply context/app/vis/callbacks.py]: Travis seems to be stalled: I'll re-run tests locally and then merge so I can continue this forward.

**Comment by mccalluc** [comment]
Travis still has a backlog problem, so I'll go ahead and merge.

**Pull request merged**